scripts/export-director-metadata

```diff
@@ -51,6 +51,12 @@ if [ ! -z ${OPSMAN_PRIVATE_KEY:+x} ]; then
   sleep 10s
 fi
 
+# Set NO_PROXY for BOSH Director and opsman host
+if [ ! -z ${SET_NO_PROXY:+x} ] && [ $SET_NO_PROXY = true ]; then
+    export NO_PROXY="${BOSH_ENVIRONMENT},$(basename "${OPSMAN_URL}")"
```

Comment:
Can you remove the `OPSMAN_URL`

Reply:
Yes, I added the `NO_PROXY` so we don't override any existing values set in the bosh deployment.